tests/test_scripts.py

```diff
@@ -180,31 +183,44 @@ def test_get_members_enableDetective_and_disableDetective():
 def test_create_members():
     d_client = Mock()
 
-    # The existing memebers and new required members don't overlap. Creation causes no errors.
+    # The existing members and new required members don't overlap. Creation causes no errors.
     d_client.create_members.return_value = {'UnprocessedAccounts': {}, 'Members': [{"AccountId": "111111111111"}, {"AccountId": "222222222222"}]}
 
-    created_members = enableDetective.create_members(d_client, "graph1", {"333333333333"}, {"111111111111": "1@gmail.com", "222222222222": "2@gmail.com"})
+    created_members = enableDetective.create_members(d_client, "graph1", False, {"333333333333"}, {"111111111111": "1@gmail.com", "222222222222": "2@gmail.com"})
```

Comment:
Should you test True in addition to False?

Reply:
See lines 211-222.